Dockerfile

```diff
@@ -0,0 +1,42 @@
+# =======================================================
+FROM node:18.12.1-bullseye-slim AS appbase
+# =======================================================
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends bzip2
+
+COPY package.json .
+
+RUN npm install
+
+# =======================================================
+FROM appbase AS development
+# =======================================================
+
```

Comment:
For development, we could also use here the development `API_URL` which is currently located in `https://palvelukartta-api-test.agw.arodevtest.hel.fi/v2/`.
`.env`-file is another option for development.